app/_how-tos/provision-dcg.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Provision a Dedicated Cloud Gateway
+content_type: how_to
+
+products:
+    - gateway
+
+works_on:
+    - konnect
+related_resources:
+  - text: Dedicated Cloud Gateways
+    url: /dedicated-cloud-gateways/
+
+tags:
+    - dedicated-cloud-gateway
+
+tldr:
+    q: How do I provision a Dedicated Cloud Gateway?
+    a: |
+      Create a Dedicated Cloud Gateway Control Plane, then configure and provision data plane nodes using the [Dedicated Cloud Gateways API](/api/konnect/cloud-gateways/v2/#/operations/create-configuration).
+
+prereqs:
+  skip_product: true
+  inline:
+    - title: Configure environment variables
+      content: |
+        Set the following variables: 
+        * `KONNECT_TOKEN`: A PAT token can be created from the [tokens](https://cloud.konghq.com/global/account/tokens) page.
+        * `KONNECT_CONTROL_PLANE_URL` = Control Plane API URL
+        For example: 
+        ```sh
+        export KONNECT_TOKEN=kpat_hUii8oWeEQWRIGuV6nnUy80A4UT6j51WHx41FLzjRodJfXbJA
+        export KONNECT_CONTROL_PLANE_URL=https://global.api.konghq.com
+        ```
+
+min_version:
+    gateway: '3.9'
+
+---
+
+## 1. Provision your Dedicated Cloud Gateway
+
+Create a Dedicated Cloud Gateway control plane:
+
+<!-- vale off -->
+{% control_plane_request %}
+url: /v2/control-planes
+status_code: 201
+method: GET
+headers:
+    - 'Authorization: Bearer $KONNECT_TOKEN'
+    - 'Content-Type: application/json'
+body:
+    name: cloud-gateway-control-plane
+    description: "A test control plane for Dedicated Cloud Gateways."
+    cluster_type: CLUSTER_TYPE_CONTROL_PLANE
+    cloud_gateway: true
+    proxy_urls:
+        - host: example.com
+          port: 443
+          protocol: https
+{% endcontrol_plane_request %}
+<!-- vale on -->
+The response body will contain a `control_plane_id`, export it as an environment variable:
+```sh
+export KONNECT_CONTROL_PLANE_ID=9c595711-84a7-4fad-b444-d089174cebe1
+```
+
+
+
+## 2. Validate Control Plane
+
+Verify that the cloud gateway network is available by sending a `GET` request to the `/cloud-gateways/network` endpoint:
```

Comment:
This was giving me a 404 as written. I _think_ we need to create the network first, but also, this example gave me a 404:

> curl -i -X POST $KONNECT_CONTROL_PLANE_URL/cloud-gateways/networks\
    -H "Authorization: Bearer $KONNECT_TOKEN"\
    -H "Content-Type: application/json"\
    --data-raw '
    {
      "name": "us-east-2 network",
      "cloud_gateway_provider_account_id": "$KONNECT_CONTROL_PLANE_ID",
      "region": "us-east-2",
      "availability_zones": [
        [
          "use2-az1",
          "use2-az2",
          "use2-az3"
        ]
      ],
      "cidr_block": "10.0.0.0/8"
    }
    '